### DISCUSSION TITLE
scroll event error

### DISCUSSION THREAD

**Discussion by kiptunor** [slint-ui/slint#3874] [Q&A]
I get this error when I use the dluent dark theme
![Screenshot 2023-11-07 172239](https://github.com/slint-ui/slint/assets/92180085/96d7657e-c454-44ae-b81f-e0867b0cd679)

how do I solve this?

**Comment by ogoffart** [comment]
scroll-event was added in 1.3 so that's not yet in the released version.
What version are you using?
You should use the nightly extension that should be in it.

**Reply by kiptunor** [reply]
I use ver 1.2.2, so version 1.3 is in development ?

**Reply by ogoffart** [reply]
Yes, version 1.3 is the development version.

**Reply by kiptunor** [reply]
I downgraded the slint source (to version 1.2.2) files for the fluent theme and I get another error involving the tab widget
![Screenshot 2023-11-07 183544](https://github.com/slint-ui/slint/assets/92180085/64aabc00-5821-4feb-af02-036c39dd90d0)

**Reply by ogoffart** [reply]
The builtin style has access to internal types that normal slint code can't access.
Similar to how the std lib in rust has access to unstable features.

What are you trying to do exactly?

**Comment by kiptunor** [comment]
I want to apply a default style when I make a build for windows.

mie., 8 nov. 2023, 06:43 Olivier Goffart ***@***.***> a scris:

**Reply by ogoffart** [reply]
You should do that in the build.rs
See https://slint.dev/releases/1.2.2/docs/slint/src/advanced/style#selecting-a-widget-style-with-rust
